firmware_mod/config/resolv.conf.dist

```diff
@@ -9,7 +9,8 @@
 ## List DNS server addresses, primary, secondary, etc
 #nameserver 192.168.1.1
 #nameserver 192.168.1.2
-#nameserver 192.168.1.3
+nameserver 1.1.1.1
```

Comment:
Not a fan of default dns servers, especially not cloudflare and google.
The average joe gets his dns server via dhcp.
The advanced users that want to setup fixed ip adresses also have to set their own dns servers.
We can keep your suggested dns servers commented out as mere suggestion.

Reply:
Commit updated - I thought this would only take effect though if a static address is assigned - but having them as suggestions works too :)